AGENTS.md

```
@@ -2,26 +2,34 @@
+## TypeScript & Imports
+- Uses strict TypeScript config (`astro/tsconfigs/strict`)
+- ES module syntax for imports
+- Type annotations where beneficial
+- No external dependencies beyond Astro core
 
-## Naming Conventions
-- Use kebab-case for file names
-- Use PascalCase for component names in imports
-- Use camelCase for JavaScript variables and functions
\ No newline at end of file
+## Terminal Portfolio Specifics
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)